pancancer_evaluation/utilities/classify_utilities.py

```diff
@@ -27,20 +27,18 @@
     NoTestSamplesError
 )
 
-def classify_cross_cancer(data_model, train_identifier, test_identifier,
-                          shuffle_labels=False, train_pancancer=False):
+def train_cross_cancer(data_model,
+                       train_gene_or_identifier,
```

Comment:
This function is called in both the `run_cross_cancer_classification.py` and `run_pan_cross_cancer_classification.py` scripts. It's a bit confusing since each of those contexts is slightly different (for the cross-cancer experiments we're training on a single gene/cancer type combination, which I'm calling an "identifier", and for the "pan-cross cancer" experiments we're training on all the data for a single gene).

The main difference in this PR is that I separated this function from the `evaluate_cross_cancer` function (before they were a single `classify_cross_cancer` function). This means I'm able to pull `train_cross_cancer` out of the inner loop in the run* scripts.

Does that make sense, or am I misunderstanding what you're asking?